renovate.json5

```diff
@@ -4,6 +4,7 @@
   "ignoreDeps": [
     // We can't always stick to latest due to infra and compatibility with other plugins
     "node",
+    "typescript",
```

Comment:
Don't want to do a comment on it like the others have?

Reply:
I think it's the same as `node`